docs/development_setup.md

```diff
@@ -10,19 +10,20 @@ In order to configure the project locally, create a copy of `.env.example` in th
 
 [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli). The Azure CLI will be used to log you in interactively.
 
-Create a virtual environment using [venv](https://docs.python.org/3/library/venv.html), [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) or [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv). 
+Install [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html). 
 
-Here is an example for setting up and activating a `venv` environment with Python 3:
+Install the required Python modules. 
 
 ```
-python3 -mvenv .venv
-source .venv/bin/activate
+. environment_setup/install_requirements.sh 
 ```
 
-Install the required Python modules in your virtual environment.
+`install_requirements.sh` create and activate a new conda environment with required Python modules.
 
 ```
-pip install -r environment_setup/requirements.txt 
+conda env create -f diabetes_regression/ci_dependencies.yml
+
+conda activate mlopspython_ci
```

Comment:
I find this a little confusing, and someone could misunderstand and run the conda commands manually.

I think ln 24 - 26 are not needed. Instead, you can provide a hyperlink to [environment_setup/install_requirements.sh](https://github.com/microsoft/MLOpsPython/blob/master/environment_setup/install_requirements.sh). Added benefit is that if this script ever changes, we won't need to update this doc.

Reply:
@j-so I deleted L24 - 26.